elasticsearch-extensions/lib/elasticsearch/extensions/test/cluster.rb

```diff
@@ -306,7 +306,7 @@ def stop
             pids = nodes['nodes'].map { |id, info| info['process']['id'] }
 
             if pids.empty?
-              return false
+              false
```

Comment:
I think you misunderstood me, here.

I meant:

- get rid of the if/else/end construct
- turn line 309 into `return false if pids.empty?` so that there is an early return